deploy/charts/kiali-operator/values.yaml

```diff
@@ -0,0 +1,59 @@
+nameOverride: ""
+fullnameOverride: ""
+
+image:
+  repo: ${$OPERATOR_QUAY_NAME}
+  tag: ${$OPERATOR_QUAY_TAG}
+  pullPolicy: Always
+  pullSecrets: []
+  # - name: xyz
```

Comment:
Should delete this superfluous line.
```suggestion
```

Reply:
Ah, I tend to do this (as do many other chart creators) to show an example of how to use that value. Happy to remove it but it seems to be a commonly trodden path. [An example here.](https://github.com/helm/charts/blob/master/stable/nginx-ingress/values.yaml#L127)